src/java/main/org/apache/jute/compiler/JType.java

```diff
@@ -27,7 +27,7 @@
 	private String mCName;
     private String mCppName;
     private String mCsharpName;
-    private String mJavaName;
+    protected String mJavaName;
```

Comment:
Haven't looked into detail here, but one observation is that our trunk and branch 3.5 is find bug free, and it does not require this private - protected change here.